app/controllers/admissions_admin/campus_controller.rb

```diff
@@ -3,7 +3,7 @@
 class AdmissionsAdmin::CampusController < AdmissionsAdmin::BaseController
   load_and_authorize_resource
 
-  def admin
+  def index
```

Comment:
Looks like some extra changes where added, these must be removed. Other than that it looks good

Reply:
I have fixed the issues:)